modules/test/playwright/tests/cookies-banner-web/cookiesBannerScript.spec.ts

```diff
@@ -0,0 +1,224 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(loginTest());
+
+test('LPD-25701 Cookie Banner Script', async ({page}) => {
+
+	// Create Site
+
+	await page.getByLabel('Open Applications MenuCtrl+Alt+A').click();
+
+	await page.getByRole('tab', {name: 'Control Panel'}).click();
+
+	await page.getByRole('menuitem', {name: 'Sites'}).click();
+
+	await page.getByRole('link', {name: 'Add Site'}).click();
+
+	await page
+		.getByRole('button', {name: 'Select Template: Blank Site'})
+		.click();
+
+	const siteName = getRandomString();
+
+	const addSiteName = await page
+		.frameLocator('iframe[title="Add Site"]')
+		.getByLabel('Name\n\n\t\t\t\n\t\t\t\t\n\n\t\t\t\tRequired');
+
+	await addSiteName.waitFor({state: 'visible'});
+
+	await addSiteName.click();
+
+	await addSiteName.fill(siteName);
+
+	await page
+		.frameLocator('iframe[title="Add Site"]')
+		.getByRole('button', {name: 'Add'})
+		.click();
+
+	await page
+		.getByText('Success: Site was successfully added.')
+		.waitFor({state: 'visible'});
+
+	await page
+		.getByRole('heading', {name: 'Site Settings'})
+		.waitFor({state: 'visible'});
+
+	const productMenuButton = await page.getByLabel('Open Product Menu');
+
+	if (await productMenuButton.isVisible()) {
+		await productMenuButton.click();
+	}
+
+	// Create Page
+
+	await page
+		.getByRole('menuitem', {name: 'Site Builder'})
+		.waitFor({state: 'visible'});
+	await page.getByRole('menuitem', {name: 'Site Builder'}).click();
+
+	await page
+		.getByRole('menuitem', {exact: true, name: 'Pages'})
+		.waitFor({state: 'visible'});
+	await page.getByRole('menuitem', {exact: true, name: 'Pages'}).click();
+
+	await page
+		.locator('li')
+		.filter({hasText: 'New'})
+		.getByRole('button')
+		.waitFor({state: 'visible'});
+	await page
+		.locator('li')
+		.filter({hasText: 'New'})
+		.getByRole('button')
+		.click();
+
+	await page
+		.getByRole('menuitem', {exact: true, name: 'Page'})
+		.waitFor({state: 'visible'});
+	await page.getByRole('menuitem', {exact: true, name: 'Page'}).click();
+
+	await page.getByRole('button', {name: 'Blank'}).waitFor({state: 'visible'});
+	await page.getByRole('button', {name: 'Blank'}).click();
+
+	const addPage = await page.frameLocator('iframe[title="Add Page"]');
+
+	await addPage.getByPlaceholder('Add Page Name').waitFor({state: 'visible'});
+	await addPage.getByPlaceholder('Add Page Name').click();
```

Comment:
To avoid duplication, you can extract these into variables.

Reply:
Code no longer needed.